RELEASE-NOTES.md

```diff
@@ -35,6 +35,8 @@ The Axom project release numbers follow [Semantic Versioning](http://semver.org/
 - Inlet: `std::shared_ptr<T>` has been replaced with `T&` in non-owning contexts
   and `std::unique_ptr<T>` in owning contexts
 - Unified core and SPIO unit tests into fewer executables to limit size of build directory
+- Updated to new BLT version that creates native CMake targets for Axom's dependencies.
+  These targets are exported behind an `axom::` namespace to avoid collisions.
```

Comment:
Should we bump the axom version for this change? It will force users to modify their build systems.

Reply:
Yes, since it's sort of an API change.

Reply:
I would not expect this change to require a user build system modification - the `axom` target will be available as usual and will work in the same way as before, but new targets will be available in the `axom::` namespace that a user's project could use if they wanted to.

That being said, I am only familiar with the build system integration of one axom user (serac), so there may be other use cases I am not considering.